README.md

```diff
@@ -79,6 +79,13 @@ From here, you can notably configure:
 
   ![Editing copybook paths in `.vscode/settings.json`](./assets/superbol-editing-copybooks-path-in-vscode-settings.png)
 
+- The copybook extensions `"superbol.cobol.copyexts"`.  
+  To configure this setting, you will need to select `Edit in settings.json`. This is
+  a list, where each entry describes a file extension if the copybook name cannot
+  be found as-is (for example `COPY "mycpy.lib"`).
+  In SuperBOL, the `default` option does not correspond to GnuCOBOL's default, but to "cpy" and
+  "cbx" only.
```

Comment:
Is GnuCOBOL's default list actually `cpy cbl cob` as hinted [here](https://github.com/OCamlPro/gnucobol/blob/fe28973b030f13b34c52d36df6b928069de21f98/cobc/cobc.c#L9077)?

Reply:
yes, first in upper, then in lower-case